akka-projection-core/src/test/scala/akka/projection/internal/metrics/OffsetCommittedCounterMetricSpec.scala

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.internal.metrics
+
+import scala.concurrent.duration._
+
+import akka.projection.HandlerRecoveryStrategy
+import akka.projection.internal.AtLeastOnce
+import akka.projection.internal.AtMostOnce
+import akka.projection.internal.ExactlyOnce
+import akka.projection.internal.FlowHandlerStrategy
+import akka.projection.internal.GroupedHandlerStrategy
+import akka.projection.internal.SingleHandlerStrategy
+import akka.projection.internal.metrics.InternalProjectionStateMetricsSpec._
+
+class OffsetCommittedCounterMetricAtLeastOnceSpec extends InternalProjectionStateMetricsSpec {
```

Comment:
Metrics have dedicated files. Each test file has 3 test suites. This makes it easy to run subsets of tests:

 - `testOnly akka.projection.internal.metrics.*AtLeastOnce*` 
 - `testOnly akka.projection.internal.metrics.OffsetCommittedCounter*`